win32ss/printing/base/winspool/printers.c

```diff
@@ -874,7 +873,6 @@ EnumPrintersW(DWORD Flags, PWSTR Name, DWORD Level, PBYTE pPrinterEnum, DWORD cb
     if (dwErrorCode == ERROR_SUCCESS)
     {
         // Replace relative offset addresses in the output by absolute pointers.
-        ASSERT(Level <= 9);
```

Comment:
These seem to assert a condition about each specific `MarshallUpStructuresArray` call, so it may make sense to keep them.

Reply:
> assert

Right, reverted. Made them consistent with the related 'Marshalling' arrays, not local `Level`s.